### PR TITLE
Add "Continuous integration"

### PR DESCRIPTION
**Pull request by stouf**
# References

- [Jisho.org entry](http://jisho.org/search/%E7%B6%99%E7%B6%9A%E7%9A%84%E3%82%A4%E3%83%B3%E3%83%86%E3%82%B0%E3%83%AC%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3)
- [Wikipedia](https://ja.wikipedia.org/wiki/%E7%B6%99%E7%B6%9A%E7%9A%84%E3%82%A4%E3%83%B3%E3%83%86%E3%82%B0%E3%83%AC%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3)

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ Front-end | フロントエンド | |
 Management | 管理 | かんり | Of people, software, etc
 Manager | 管理者 | かんりしゃ |
 Infrastructure | インフラ | |
+Continuous integration | 継続的インテグレーション | けいぞくてきいんてぐれーしょん |
```
Comment (ronkorving) [inline ReadMe.md]: In the `Kana` field, please keep the `いんてぐれーしょん`-part katakana. It's possible my approach is counter-intuitive to some people, but that's how I've put the words in so far. If you think that's a bad approach, feel free to open an issue and we can discuss that there.

**Comment by ronkorving** [comment]
Thanks! I'll have it double checked, and would like to wait for a new PR from @beaucollins who is adding links to jisho.org on all words. See the comments on #7

**Comment by ronkorving** [comment]
Double check: confirmed. LGTM (just waiting for #7).

File: ReadMe.md (hunk 1 above)
Comment (stouf) [reply ReadMe.md]: Nop, it sounds good to me and it's probably more readable too :) I will update.

Comment (stouf) [reply ReadMe.md]: Done

**Comment by AlmirKadric** [comment]
LGTM 👍

**Pull request merged**